packages/api/src/models/cleaningTaskModel.js

```diff
@@ -45,36 +45,27 @@ class CleaningTaskModel extends Model {
     json = super.$formatDatabaseJson(json);
     const weightUnits = ['g', 'lb', 'kg', 't', 'mt', 'oz'];
     const volumeUnits = ['l', 'gal', 'ml', 'fl-oz'];
-    const defaultAction = (json) => {
-      json.volume = json.product_quantity;
-      //Database previously defaulted to 'l'
-      json.volume_unit = 'l';
-      json.weight = null;
-      json.weight_unit = null;
-      delete json.product_quantity;
-      delete json.product_quantity_unit;
-    };
-    if (json.product_quantity && json.product_quantity_unit) {
-      if (weightUnits.includes(json.product_quantity_unit)) {
+    if (json.product_quantity) {
+      if (json.product_quantity_unit && weightUnits.includes(json.product_quantity_unit)) {
         json.weight = json.product_quantity;
         json.weight_unit = json.product_quantity_unit;
         json.volume = null;
         json.volume_unit = null;
-        delete json.product_quantity;
-        delete json.product_quantity_unit;
-      } else if (volumeUnits.includes(json.product_quantity_unit)) {
+      } else if (json.product_quantity_unit && volumeUnits.includes(json.product_quantity_unit)) {
```

Comment:
I came back to check something 😂 
If you can change this to `else`, you can do:
`json.volume_unit = json.product_quantity_unit || 'l';`

But this is just my preference!

Reply:
Haha nice, it should all be deleted in LF4263 anyways -- going to move it through!